src/routes/resources/+page.svelte

```diff
@@ -86,6 +86,23 @@
 			<a href="https://github.com/janosh/awesome-sveltekit" target="_blank">awesome-sveltekit</a>
 		</li>
 	</ul>
+
+	<h2>UI frameworks</h2>
+	<div>Check out the following list of Svelte UI component libraries:</div>
+	<ul>
```

Comment:
these are all listed on https://sveltesociety.dev/components#design-systems. we shouldn't partially duplicate the list here